salt/states/lvm.py

```diff
@@ -126,7 +126,7 @@ def vg_present(name, devices=None, **kwargs):
         ret['comment'] = 'Volume Group {0} already present'.format(name)
         for device in devices.split(','):
             realdev = os.path.realpath(device)
-            pvs = __salt__['lvm.pvdisplay'](realdev)
+            pvs = __salt__['lvm.pvdisplay'](realdev, real=True)
             if pvs and pvs.get(realdev, None):
                 if pvs[realdev]['Volume Group Name'] == name:
                     ret['comment'] = '{0}\n{1}'.format(
```

Comment:
would it hurt to keep the old behaviour of extending the vg with the device-name rather than with the real-name(like vgcreate does)
change line 136:
from
```
__salt__['lvm.vgextend'](name, realdev)
```
to
```
__salt__['lvm.vgextend'](name, device)
```